qiskit/unrollers/_jsonbackend.py

```diff
@@ -34,9 +34,8 @@
         ]
     }
 """
-from qiskit.unrollers._backenderror import BackendError
-from qiskit.unrollers._unrollerbackend import UnrollerBackend
-from qiskit import QISKitError
+
+from qiskit.unroll import BackendError, UnrollerBackend
```

Comment:
This looks like a rebase issue. The imports here should be either what was already there in L37-L39 or changed to be an absolute import `from qiskit.unrollers`. Importing from qiskit.unroll like this will cause pylint to complain about a circular import because the import path looks like `qiskit.unrollers._jsonbackend` -> `qiskit.unroll` -> `qiskit.unrollers._jsonbackend`.